.ruff.toml

```diff
@@ -0,0 +1,33 @@
+fix = true
+indent-width = 4
+line-length = 120
```

Comment:
I know this isn't the time to litigate this, but at some point I'd like to discuss dropping this to 80 or Black's default of 88, since 120 often isn't great for reviewing PRs or editing horizontal code panes on a 14-16" laptop.

A discussion for the future! 😉

Reply:
Yeah happy to discuss offline! Not super attached to this